### DISCUSSION TITLE
请问qanything2.0的源代码没有完全公布吗

### DISCUSSION THREAD

**Discussion by zmscc** [netease-youdao/QAnything#579] [Q&A]
![image](https://github.com/user-attachments/assets/054d1a03-b04a-4a21-837b-c41dc4fd3c2e)

**Comment by TatyOko28** [comment]
From the screenshot, it appears that certain parts of the code might be commented out or omitted in the repository. For example, the line:
```bash
# asyncio.create_task(local_doc_qa.insert_files_to_milvus(user_id, kb_id, local_files))
```
suggests that some functionality might not be fully disclosed or intentionally disabled.

This could mean that:
The full implementation (e.g., local_doc_qa.insert_files_to_milvus) is either not provided or is located in a different private or third-party module.
The source code may be partially open but requires further clarification from the maintainers.
